packages/fast-components-react-base/src/tabs/tabs.props.ts

```diff
@@ -5,6 +5,22 @@ import {
 } from "@microsoft/fast-components-class-name-contracts-base";
 import { Orientation } from "@microsoft/fast-web-utilities";
 
+export interface TabConfig {
+    className: string;
+    key: string;
+    "aria-controls": string;
+    active: boolean;
+    onClick: (e: React.MouseEvent) => void;
+    onKeyDown: (e: React.KeyboardEvent) => void;
+    tabIndex: number;
+}
+
+export interface Item {
```

Comment:
Can we arrive at a more illustrative name for this?

Reply:
+1

Reply:
@janechu @chrisdholt and I chatted about this pretty extensively. `TabItem` is already being used. I originally had `tabItemData` but @janechu asked me to remove data. You Can see  Jane's prevuous comment.